loopsPartTwo.js

```diff
@@ -133,6 +133,12 @@ console.log(result[0] == 0 && result[1] == 10);
 
 function convertRoutineFromNewFormat(routineString) {
     let routine = [];
+    routine = routineString.split("|")
+      for ( let set of routine){
+          set = set.split(:);
+          const numOfReps = setArray[];
```

Comment:
setArray isn't defined, is this supposed to be set?
Or is the previous line supposed to be 
setArray = set.split(":");
(Notice the parenthesis around the colon

Reply:
The missing quotes around the color are what is breaking your file :)